src/index.html

```diff
@@ -5,8 +5,247 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="moyo_header">
+      <nav>
+        <div class="header">
+          <a href="#" class="home header__home">
+            <img
+              src="images/logo.png"
+              alt="logo"
+              class="home__img"
+            >
+          </a>
+          <div class="links header__links">
+            <a href="#apple" class="links__link is_active">APPLE</a>
+            <a href="#samsung" class="links__link">SAMSUNG</a>
+            <a href="#s_and_ph" class="links__link">СМАРТФОНИ И ТЕЛЕФОНИ</a>
+            <a
+              href="#lp_and_pc"
+              class="links__link"
+              data-qa="hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+            <a href="#gad" class="links__link">ГАДЖЕТЫ</a>
+            <a href="#pln" class="links__link">ПЛАНШЕТЫ</a>
+            <a href="#photo" class="links__link">ФОТО</a>
+            <a href="#video" class="links__link">ВИДЕО</a>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main class="page">
+      <div class="page__first-block">
+        <div class="product-card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="imac"
+            class="product-card__photo"
+          >
+          <h1 class="model-name product-card__model-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+          <section class="model-code product-card__model-code">Код товара: 195434</section>
+          <section class="reviews product-card__reviews">
+            <div class="stars reviews__stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <article class="text-reviews reviews__text-reviews">Отзывов: 5</article>
+          </section>
+          <section class="price product-card__price">
+            <article class="price-text price__price-text">Цена:</article>
+            <article class="amount price__amount">69 999 грн</article>
+          </section>
+          <a class="button product-card__button" data-qa="hover">КУПИТЬ</a>
+        </div>
+
+        <div class="product-card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="imac"
+            class="product-card__photo"
+          >
+          <h1 class="model-name product-card__model-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+          <section class="model-code product-card__model-code">Код товара: 195434</section>
```

Comment:
Don't make separate block for each piece of text. Make them elements of `product-card` block.